tests/testthat/test-drive_mkdir.R

```diff
@@ -0,0 +1,29 @@
+context("Make folders")
+
+## NOTE if you do not currently have the files needed,
+## change run & clean below to TRUE to create files needed
+## (CAUTION, this will delete files that will interfere)
+
+nm_ <- nm_fun("-TEST-drive-mkdir")
+
+run <- FALSE
+clean <- FALSE
+if (run) {
+  ## make sure directory is clean
+  if (clean) {
+    del <- drive_delete(c(nm_("foo"), nm_("bar")))
+  }
+  drive_mkdir(nm_("foo"))
+}
+
+test_that("drive_mkdir() places folder in the correct folder", {
+
+  foo <- drive_path(nm_("foo"))
+  ## foo
+  ## |- bar
+  bar <- drive_mkdir(nm_("bar"), path = nm_("foo"))
+  expect_identical(bar$files_resource[[1]]$parents[[1]], foo$id)
+
+  ## clean up
+  drive_delete(nm_("bar"))
+})
```

Comment:
yes! Sorry I deleted my previous comment 🙈 - the last commit does just that 👆. The test itself just deletes the parent folder, but the clean up checks both just in case something funky happens.